fiasko_bro/validators.py

```diff
@@ -475,3 +475,11 @@ def is_nesting_too_deep(solution_repo, tab_size, max_indentation_level, whitelis
                     file_name = url_helpers.get_filename_from_path(file_path)
                     return 'too_nested', '{}:{}'.format(file_name, line_number)
             previous_line_indent = indentation_spaces_amount
+
+
+def has_no_string_sums(solution_repo, *args, **kwargs):
```

Comment:
It would be better to call it `has_no_string_literal_sums`, since it does not catch the case in which two string variables are summed up (and it's ok, the `ast` module doesn't have the type inference capabilities).